src/item_pocket.cpp

```diff
@@ -504,6 +504,16 @@ units::mass item_pocket::item_weight_modifier() const
     return total_mass;
 }
 
+units::length item_pocket::item_length_modifier() const
+{
+    units::length total_length = 0_mm;
+    for( const item &it : contents ) {
+        total_length = std::max( static_cast<units::length>( it.length() * data->volume_multiplier ),
```

Comment:
Should this be multiplying by the cube root of the volume multiplier instead?

Reply:
...Right, elementary mistake